src/js/eva-config.js

```diff
@@ -75,6 +75,18 @@ function deDuplicatedSpeciesList(speciesList) {
     });
 }
 
+function deDuplicateSpeciesListWithAssemblyAccession(speciesList) {
+    //Show multiple assemblies within a single build in the same species as opposed to deDuplicatedSpeciesList
+    //which only shows only one entry per species ex: Human only shows grch37 and grch38
+    //ex: Human should show grch38.p3, grch38.p12 etc.,
+    speciesList = _.sortBy(speciesList, function(listItem) {
+                                            return [listItem.assemblyAccession,
+                                                    listItem.taxonomyEvaName.toLowerCase()].join(); });
+    return _.uniq(speciesList, function(listItem, key, unused) {
```

Comment:
oh, I forgot: what prototype is this `_.uniq`? is it [underscore's uniq](https://underscorejs.org/#uniq)? are you sure it's doing what you want? maybe it is asking for a boolean and `(bool)function (...) {...}` returns true?

Reply:
This is underscore's uniq and has the same pattern as used in the function above which was tested to work with the assemblyCode de-duplication.